static/src/stylesheets/module/content/live-blog/navigation.scss

```diff
@@ -42,82 +39,103 @@
 
 .liveblog-navigation__link {
     display: inline-block;
+    position: relative;
+    color: colour(neutral-1);
+    background: transparent;
+    border: 1px solid colour(neutral-4);
+    height: $gs-baseline * 3;
+    line-height: 38px;
+    font-weight: bold;
+    vertical-align: middle;
 
-    // Override user-agent styles for <a>
     &:hover {
+        border-color: colour(neutral-2);
         text-decoration: none;
     }
 
-    .inline-icon {
-        vertical-align: middle;
-        display: inline-block;
+    svg {
+        fill: colour(neutral-2);
+        position: absolute;
     }
 
-    svg {
-        display: block;
+    @include mq($until: mobileLandscape) {
+        font-size: 0;
     }
 }
 
 .liveblog-navigation__link--primary {
-    padding: $gs-gutter / 2;
-    background: colour(guardian-brand-dark);
+    width: $gs-baseline * 3;
     border-radius: 100%;
 
-    .inline-icon {
-        fill: #ffffff;
+    svg {
+        top: 10px;
+        width: 16px;
+        height: 16px;
     }
 
-    svg {
-        width: 21px;
-        height: 21px;
+    .inline-chevron-right svg {
+        right: 9px;
     }
-}
 
-.liveblog-navigation__link--primary--disabled {
-    background: colour(neutral-5);
+    .inline-arrow-left svg {
+        left: 9px;
+    }
 }
 
 .link-navigation__link--secondary {
-    // Increase target area
-    padding-top: $gs-gutter / 2;
-    padding-bottom: $gs-gutter / 2;
-    // Override user-agent styles for <a>
-    color: inherit;
-    font-weight: bold;
-
-    .inline-icon {
-        fill: currentColor;
-    }
+    border-radius: 21px;
+    padding: 0 $gs-gutter / 2;
 
     svg {
-        width: 13px;
-        height: 13px;
+        width: 14px;
+        height: 14px;
+        top: 11px;
     }
-}
 
-.link-navigation__link--secondary--newer {
-    margin-right: $gs-gutter / 4;
-    @include mq(mobileLandscape) {
-        margin-right: $gs-gutter / 2;
+    &--newer {
```

Comment:
In the past we've avoided this syntax because it breaks searchability, i.e. you can no longer do a project wide search for `link-navigation__link--secondary--newer` and yield all the results.